netbox/netbox/api.py

```diff
@@ -4,11 +4,22 @@
 from rest_framework.pagination import LimitOffsetPagination
 from rest_framework.permissions import DjangoObjectPermissions, SAFE_METHODS
 from rest_framework.renderers import BrowsableAPIRenderer
+from rest_framework.schemas import coreapi
 from rest_framework.utils import formatting
 
 from users.models import Token
 
 
+def is_custom_action(action):
+    return action not in {
+        'retrieve', 'list', 'create', 'update', 'partial_update', 'destroy', 'bulk_destroy'
+    }
```

Comment:
Wonder if it would be better to do something like:

```python
from restframework.schemas.coreapi import is_custom_action as base_is_custom_action

def is_custom_action(action):
    return base_is_custom_action(action) and action not in {'bulk_destroy'}
```

So that we are clearly extending the base function, just in case it ever changes?